src/sqlfluff/core/rules/base.py

```diff
@@ -1177,7 +1189,27 @@ def get_rulelist(self, config) -> List[BaseRule]:
                 else:
                     alias_map[alias].append(manifest.code)
         # Incorporate after all checks are done.
-        reference_map = {**alias_map, **reference_map}
+        return {**alias_map, **reference_map}
+
+    def get_rulepack(self, config) -> RulePack:
+        """Use the config to return the appropriate rules.
+
+        We use the config both for allowlisting and denylisting, but also
+        for configuring the rules given the given config.
+
+        Returns:
+            :obj:`list` of instantiated :obj:`BaseRule`.
```

Comment:
Needs updating (or removing)... now returns `RulePack`

Reply:
I'll just remove it now - I think the type hints are enough.